### DISCUSSION TITLE
Guidance Regarding GSOC'24

### DISCUSSION THREAD

**Discussion by skushagra9** [ruxailab/RUXAILAB#305] [General]
Dear enthusiasts, first of all, I want to thank your interest in our small organization.
As you have seen we have been working during the last 5 years on different things but all of them are related to the creation of usability systems that help practitioners of UX to perform their activities.

My suggestion is that you start creating a fork of the project that you think that are most interesting for you and start collaborating in the way that you consider best. You will probably see repositories with few documentation so maybe starting with a pull request helping on that might be awesome!

I will use this thread in order to let you present, and give tips about where can you start.

Sincerely,

M.

**Comment by praptisharma28** [comment]
Dear Sir/Ma’am,

I hope this message finds you well. I do full stack web development and open source contributions and I'm eager to contribute to your GitHub repositories and projects as well. My technology stack include HTML, CSS, Django, JavaScript, SQL (MySQL/NoSQL), Python, C, C++. I am also comfortable with MERN stack. Could you direct me to repositories needing these skills or suggest specific tasks for contribution? I'm enthusiastic about collaborating and adding value to your project- Data Extraction for Sentiment Analysis from Usability Tests.

Thank you,
Prapti Sharma
B.tech. 2nd year student
Github Username: @praptisharma28

**Comment by imshivam8** [comment]
Hello! I’m Shivam Kumar, a pre-final year student at Guru Tegh Bahadur Institute of Technology, where I am pursuing a degree in Electronics and Communication. My passion lies in contributing to open-source projects, and I am eager to participate in Google Summer of Code (GSoC) 2024. My technical expertise includes Java, C++, data structures, algorithms, and web development.
i am interested in 
1.Integration of Eye Tracking with Accessibility Features (90h)
please guide in this project

**Comment by opxdelwin** [comment]
Hello maintainers,

Myself a passionate 2nd year student at KIIT UNIVERSITY, looking to contribute to this project under GSoC '24.

Upon having a look into this year's ideas, and checking out the technologies used, could you please confirm if there's no Flutter project for this year? Flutter was mentioned in repositories' README.md. It would be so helpful if you could confirm the same!

Thanks,
Delwin

**Comment by marcgc21** [comment]
Dear enthusiasts, first of all, I want to thank your interest in our small organization.
As you have seen we have been working during the last 5 years on different things but all of them are related to the creation of usability systems that help practitioners of UX to perform their activities.

My suggestion is that you start creating a fork of the project that you think that are most interesting for you and start collaborating in the way that you consider best. You will probably see repositories with few documentation so maybe starting with a pull request helping on that might be awesome!

Sincerely,

M.

**Comment by marcgc21** [comment]
> Hi, I am intereseted in these projects: Eye Tracking Algorithm Optimization Eye Tracking Data Visualization Tool How should I get started ??

Regarding this topic; I suggest you to create a fork of the repository and starting checking how the project works to see what can you do to improve it!

**Comment by Anneqxy** [comment]
Hello, I am Anne(Xiangyue Que). I am a second-year student in University of Illinois Urbana-Champaign. My major is Computer Engineering. In the past two years in UIUC, I learned a lot of skills, such as Python, C++, Java. I used them to solve a lot of machine problems. I also built several websites with my friends, which allowed me to learn Javascript, CSS, HTML. I want to join "Eye Tracking Data Visualization Tool" group. Could you guide me how to apply it and begin working on it? Thank you!
[Anne resume.pdf](https://github.com/ruxailab/RUXAILAB/files/14391821/Anne.resume.pdf)

**Comment by BasmaElhoseny01** [comment]
Hello I am Basma Elhoseny a 4th year computer Engineering student at Cairo University Faculty of Engineering. I am interested in Sentiment Analysis Data Extraction from Usability Tests (350h) and Integration of heat maps into Remote Usability LAB. (175h) . I have been working for Web develoment for abot 2 years and recently i am working on GP which is mainly based on NN and ML techniques. I am good at learning NN technqiues. I have worked on several ML project like Hand Geature recognito which is baed on pattern recogntionnhttps://github.com/BasmaElhoseny01/Hand-Gesture-Recognition. We have sucessfully achieved an acc of 73% on the test data set provoded by our mentors to be ranked as 3rd on data board. My First NN poject was in NLP filed which is Tashkeel (Arabic Diacritization System). https://github.com/BasmaElhoseny01/Tashkeel we have achieved 97% acc on the test data set provided by our TA . I am also experienced in UI/UX design .
github: https://github.com/BasmaElhoseny01
Behance: https://www.behance.net/basmahelhosen

**Comment by pavinanaicker** [comment]
Dear Sir/Ma’am,
 I am reaching out to express my interest in contributing to your project, the Eye Tracking Data Visualization Tool, as listed on your GitHub repository.
As a fourth-year undergraduate student specializing in Information Technology with an Honors focus in Data Science, I am deeply passionate about projects that involve data visualization and analysis. The prospect of working on an eye tracking data visualization tool particularly excites me due to its interdisciplinary nature and potential impact across various fields.

My skill set encompasses a range of technologies including Python, JavaScript, HTML, CSS, and frameworks such as Django and React. Moreover, my expertise in data science equips me with the necessary knowledge to handle complex data sets and implement effective visualization techniques.

I am eager to contribute to your project by leveraging my skills and knowledge. Could you please provide me with more information on how I can get involved and contribute meaningfully to the Eye Tracking Data Visualization Tool project?

Thank you for considering my interest in your project. I am looking forward to the possibility of collaborating with your team and contributing to this exciting endeavor.

Best regards,
Pavina Naicker
Fourth-year Undergraduate Student
GitHub Username: pavinanaicker

**Comment by GiteshDewangan** [comment]
Hello @skushagra9 sir,

I'm Giteshwar Dewangan, a pre-final year student at SSTC BHILAI. Fueled by curiosity in tech, I've delved into MERN stack projects and explored JavaScript, TypeScript, and React at Internsavy. During my internship, I crafted portfolios and contact projects.

My enthusiasm extends beyond GSoC participation; I'm genuinely keen on making substantial contributions to your project. I aim to create a positive impact, enhancing my experience and learning along the way.

I would greatly appreciate your guidance and support throughout this exciting opportunity.

Best regards,
Giteshwar Dewangan

**Comment by ogechi3** [comment]
Dear Sir / madam I am Thomas Jennifer, I am writing to express my interest in contributing to your project "Integration of heat maps into remote usability LAB"

My skill set includes a range ot technologies including HTML, CSS, JAVASCRIPT AND REACT,  I have worked with these languages to create beautiful projects.

I hope to get guidance from you on how to apply and make an impact, I look forward to your response.Thank you 

Best regards 
Thomas Jennifer

**Comment by pankaj87072** [comment]
hey @marcgc21 @jvJUCA Is this org is in gsoc 2023 or 2022 because as i am searching it in org list because of some reference from previous contributers projects , not able to get but the idea list is present so can you tell if this org participated or not??

**Reply by KarinePistili** [reply]
Hi @pankaj87072 ! Thank you for the interest in or org! This is the first time we are participating in GSoC! To get started I suggest you to fork the repositories you are interested on, try them on and if you have any suggestions to improve it, present the ideas on a pull request or discussion :)

**Reply by pankaj87072** [reply]
@KarinePistili thanks for the help

**Comment by BasmaElhoseny01** [comment]
Hello I am Basma Elhoseny a 4th year computer Engineering student at Cairo University Faculty of Engineering. I am interested in Sentiment Analysis Data Extraction from Usability Tests (350h) and Integration of heat maps into Remote Usability LAB. (175h) . I have been working for Web develoment for abot 2 years and recently i am working on GP which is mainly based on NN and ML techniques. I am good at learning NN technqiues. I have worked on several ML project like Hand Geature recognito which is baed on pattern recogntionnhttps://github.com/BasmaElhoseny01/Hand-Gesture-Recognition. We have sucessfully achieved an acc of 73% on the test data set provoded by our mentors to be ranked as 3rd on data board. My First NN poject was in NLP filed which is Tashkeel (Arabic Diacritization System). https://github.com/BasmaElhoseny01/Tashkeel we have achieved 97% acc on the test data set provided by our TA . I am also experienced in UI/UX design .
github: https://github.com/BasmaElhoseny01
Behance: https://www.behance.net/basmahelhosen @KarinePistili

**Comment by tahamukhtar20** [comment]
Hello everyone, I'm Taha, a 3rd year CS major from Pakistan. Nice to meet y'all. I'm interested in the **Integration of Eye Tracking with Accessibility Features** project and am looking forward to working with you. 
Here is a link to my resume: https://drive.google.com/file/d/1D-qL1VVqG23kt9Skgb0qRRDqRu2ShkLa/view?usp=sharing

**Comment by vishisht-dubey** [comment]
Hello Team,
My name is Vishisht Dubey, a 3rd year undergrad at Indian Institute of Technology, Banaras Hindu University, Varanasi, I have been learning about different tools and technologies since past 1.5 years, i have significant experience with technologies like html, css, javascript, nextjs, typescript, react-redux, django, python, firebase, aws, docker and i have completed 3 internships till now. Other than this, I am also the part of the tech team of Training and Placement Cell, IIT-BHU and E-Cell IIT-BHU.
I had a keen interests in software building and i am interested in the project `Eye Tracking Algorithm Optimization (350h)` as it matches with the knowledge i had and perfectly align with my skills and i would be able to contribute much better in that project. Here's my resume [link](https://drive.google.com/file/d/1Ig2b_3g1FWqbgbw0ZpjfmKuzvnDPMH1l/view?usp=sharing) Looking forward to working with the mentors. Thank you.

**Comment by Jaishree2310** [comment]
Hi! I'm Jaishree Singh, a JavaScript and web developer with a passion for optimizing software to run efficiently and swiftly. Although my expertise primarily lies in front-end development, I've diversified my skill set to include back-end technologies and desktop app development, leveraging MongoDB for database management, Python for server-side logic, and Firebase for real-time data handling. My journey in the tech world has been greatly enriched by contributing to and learning from the open-source community. I've shared my projects on GitHub and continuously seek to learn the best coding practices. Working with technologies like Vue.js has honed my front-end development skills, allowing me to create dynamic and responsive user interfaces. I'm eager to collaborate with others who share a dedication to advancing technology through open-source contributions.

**Comment by KarinePistili** [comment]
Hello contributors! Please refer to discussion #306 to introduce yourselves. Guidance about the process of contributing and sending a proposal will be announced on "Announcements" discussions.
